### PR TITLE
Milestone 8: Popup Window

### PR DESCRIPTION
**Pull request by SaidRasinlic**
In this milestone we've done the following things:

- Created mobile popup
- Created desktop popup
- Added responsive buttons

### CODE REVIEW

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @SaidRasinlic , 

### Good job so far!
  - ✔ Articles are dynamically generated.
  - ✔ Popup works as expected.
  - ✔ No linter errors.


#### There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [ ] Good job following the template for the popup window, however, at a certain point between the mobile and desktop view, the button links are broken trying to fit in the space. I suggest you create a brekpoint and display them as column or give the container a fixed width to fix the problem, please check it out.


     


_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ ] [OPTIONAL] At the same breakpoint mentioned above, the cards break on the desktop view, showing the button outside of the container and breaking the cards. I suggest you fix this issue to make your portfolio look more professional!


     



Cheers and Happy coding! 👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by SaidRasinlic** [comment]
Fixed everything 😃

**Review by medaminedev66 — APPROVED** [review]
# Hi @SaidRasinlic , 

### Status: Approved ✔️

:+1: You have implemented all the suggestions. Well Done.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions

-  Please also consider writing your commit messages in a professional way as that makes your repo more professional. To see how you can make your work more professional you can visit this [link](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md)

**You may close this Pull Request and merge the changes**

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Amine Smahi_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/main.js [hunk 1]
```diff
@@ -6,5 +6,139 @@ const displayMobileMenu = () => {
...
+  technologies: ['html', 'Ruby on rails', 'css', 'javascript'],
+  popupTech: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+  image: './images/uber-nav.svg',
+  imageMobile: './images/multi-portfolio-two.svg',
+  imagePopup: './images/uber-nav.svg',
+  imageMobilePopup: './images/multi-portfolio-two.svg',
+  seeLive: 'https://saidrasinlic.github.io/MicroversePortfolio/',
+  seeSource: 'https://github.com/SaidRasinlic/MicroversePortfolio',
+},
+];
```
Comment (medaminedev66) [inline js/main.js:70]: - [OPTIONAL]
All the popups shows the same content `Tonic`. While I see in your `projectDetail` table other contents (ex: 'Facebook 360',  'Uber Navigation'). Kindly configure your pop-up content so it displays the content of the clicked project.

**Pull request merged**